tensorflow/lite/micro/kernels/cmsis-nn/conv.cc

```diff
@@ -268,9 +268,6 @@ TfLiteStatus EvalQuantizedPerChannel(
              (output_width % 4 == 0) && batches == 1) {
     const int32_t buf_size = arm_convolve_1_x_n_s8_get_buffer_size(
         input_depth, filter_width, filter_height);
-    if (get_cmsis_scratch_buffer(context, &buf, buf_size) != kTfLiteOk) {
-      return kTfLiteError;
-    }
     if (arm_convolve_1_x_n_s8(
```

Comment:
This is also fixed (along with some other stuff) in https://github.com/tensorflow/tensorflow/pull/38123

Reply:
SG, I assume it's fine to leave it in here since it'll just merge out in the end.